content/en/container-orchestration.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Container Orchestration
+status: Completed
+category: Concept
+---
+
+## What it is
+Container orchestration automates container deployment, management, scaling, and networking. It's primary goal is to manage the life cycle of containers and their dynamic environments.A conductor, like in an orchestra, directs a group of musicians who work collaboratively to perform a piece of music. Similarly containers in your application are orchestra musicians.
+
+## Problem it addresses 
+On behalf of the user, container orchestration organises the compute, networking, and storage infrastructure. To ensure that applications are highly available and computing resources are used to their full potential, it does away with the requirement for direct manual orchestration in a cluster and automates the orchestration process.  
```

Comment:
I think that currently most words in the CNCF Glossary are in American style.
And I checked [Style Guide](https://glossary.cncf.io/style-guide/), but cannot find sentence that mentions about spelling style (maybe we need this now or later).

Dear maintainers (@CathPag @JasonMorgan @iamNoah1 @seokho-son),
what do you think about this? 😊

```suggestion
On behalf of the user, container orchestration organizes the compute, networking, and storage infrastructure. To ensure that applications are highly available and computing resources are used to their full potential, it does away with the requirement for direct manual orchestration in a cluster and automates the orchestration process.  
```